<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Card UID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>floor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regiButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Company</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="uidEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="floorEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="companyEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="isRegistered">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>등록</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>카드 정보</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>등록 여부</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>UID</string>
    </property>
   </widget>
   <widget class="QLabel" name="uidInfo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>uid num</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
